File: TesteTecnicoFrontend/src/app/modelos/Livro.ts
export class Livro {
  id: string;
  titulo: string;
  autor: string;
  genero: string;
  anoPublicacao: number;

  constructor(id: string, titulo: string, autor: string, genero: string, anoPublicacao: number) {
    this.id = id;
    this.titulo = titulo;
    this.autor = autor;
    this.genero = genero;
    this.anoPublicacao = anoPublicacao;
  }
}

File: TesteTecnico/Models/Livro.cs
﻿namespace TesteTecnico.Models
{
    public class Livro
    {
        public Guid Id { get; set; }
        public string Titulo { get; set; }
        public string Autor { get; set; }
        public string Genero { get; set; }
        public int AnoPublicacao { get; set; }
    }
}
